examples/restaurant_demo/restaurant.py

```diff
@@ -0,0 +1,341 @@
+import pydp as dp
+from pydp.algorithms.laplacian import BoundedSum, Count
+
+import math
+import statistics as s
+import pandas as pd
+from collections import defaultdict
+
+# An hour when visitors start entering the restaurant (900 represents 9:00 AM)
+OPENING_HOUR = 900
+
+# An hour when visitors stop entering the restaurant (2000 represents 20:00 PM)
+CLOSING_HOUR = 2000
+
+# Range of valid work hours when a visitor can enter the restaurant.
+VALID_HOURS = list(range(OPENING_HOUR, CLOSING_HOUR + 1))
+
+# For how many hours visitors can enter the restaurant
+NUM_OF_WORK_HOURS = CLOSING_HOUR - OPENING_HOUR + 1
+
+# Number of visit days contributed by a single visitor will be limited to 3. All exceeding visits will be discarded.
+COUNT_MAX_CONTRIBUTED_DAYS = 3
+
+# Number of visit days contributed by a single visitor will be limited to 4. All exceeding visits will be discarded.
```

Comment:
Question - can you help me understand what the difference between COUNT and SUM in the context of CONTRIBUTED DAYS is? 

I'm thinking:

```diff
- # Number of visit days contributed by a single visitor will be limited to 4. All exceeding visits will be discarded.
+ # Cap the maximum number of visiting days at 4 per each visitor (any number above will not be taken into account)
```

But we should differentiate the max. at 3 and 4 here.

Reply:
There's not a difference besides replicating the code from the original demo, they used different variables so I wrote both of them here as well.